### ISSUE TITLE
Activities are not available in SwiftPM projects (even on macOS) when using `swift test`

### ISSUE THREAD

**Issue by orta** [pointfreeco/swift-snapshot-testing#121]
Hey folks, ran out of my timebox for this one, but I'm running on macOS, but using SwiftPM. This means one of the `XCTContext.runActivity` is running, but the closed-source XCTest isn't available. This means `#if !os(Linux)` isn't going to work when running via `swift test` . 

[Their check](https://github.com/apple/swift/blob/master/stdlib/public/SDK/XCTest/XCTest.swift#L35) probably isn't available for us to use, if you want to test it out:

```sh
git clone https://github.com/danger/danger-swift.git
cd danger-swift
git checkout snapshots # or dc41e4e06eb2c994e0ece6fdc355386a212fc0fd
swift build
swift run komondor install
swift package generate-xcodeproj
open Danger.xcodeproj

swift test
```

**Comment by mbrandonw** [comment]
Ha perfect :/ I'll look into this tonight or first thing in the morning.

Thanks for the reports!

**Comment by orta** [comment]
@briancroom any idea how we can detect if the XCTest runtime is the swift OSS version and doesn't have runActivity?

**Comment by stephencelis** [comment]
This shouldn't have been auto-closed, sorry!

**Comment by stephencelis** [comment]
@orta Got a fix here: #143. Not the loveliest solution but seems to work.

**Comment by orta** [comment]
Thanks folks, will give it a re-run today 👍
